x/ibc-hooks/ibc_middleware_test.go

```diff
@@ -3,9 +3,10 @@ package ibc_hooks_test
 import (
 	"encoding/json"
 	"fmt"
-	ibc_hooks "github.com/osmosis-labs/osmosis/v13/x/ibc-hooks"
 	"testing"
 
+	ibc_hooks "github.com/osmosis-labs/osmosis/v13/x/ibc-hooks"
```

Comment:
(unimportant human lint) should this be ibchooks for consistency with app?

Reply:
changed